### DISCUSSION TITLE
FATAL: Authentication error, reason: "Invalid format for user or db_name"

### DISCUSSION THREAD

**Discussion by X-Titouan** [prisma/prisma#25394] [Q&A]
### Bug description

Hey :)

I have this error:

```dotenv -e .env.local npx prisma migrate dev

Prisma schema loaded from src/prisma/schema.prisma
Datasource "db": PostgreSQL database "postgres", schema "public" at "aws-0-eu-west-3.pooler.supabase.com:5432"

Error: FATAL: Authentication error, reason: "Invalid format for user or db_name"
   0: schema_core::state::DevDiagnostic
             at schema-engine/core/src/state.rs:276
```

This is my .env.local:

```
NODE_ENV="development" 

DATABASE_URL="postgresql://postgres.jobaiwpgsttytymjmgep:[ALPHA_NUMERIC]@aws-0-eu-west-3.pooler.supabase.com:6543/postgres?pgbouncer=true&connection_limit=1"
DIRECT_URL="postgresql://postgres.jobaiwpgsttytymjmgep:[ALPHA_NUMERIC]@aws-0-eu-west-3.pooler.supabase.com:5432/postgres"
```

I have found only this SOF post:

https://stackoverflow.com/questions/78993399/prisma-supabase-invalid-format-for-user-or-db-name-after-initial-migration

But I don't want to change my database location...

Any ideas ?

### Prisma information

```
generator client {
    provider = "prisma-client-js"
}

generator dbml {
    provider = "prisma-dbml-generator"
}

datasource db {
    provider  = "postgresql"
    url       = env("DATABASE_URL")
    directUrl = env("DIRECT_URL")
}

model User {
    id        String   @id @default(uuid())
    email     String   @unique
    updatedAt DateTime @updatedAt
    createdAt DateTime @default(now())
}
```

### Environment & setup

- OS: Ubuntu
- Database: Supabase
- Node.js version: v20.12.2

### Prisma Version

```
"prisma": "^5.20.0",
```

**Comment by ludralph** [comment]
Hi @X-Titouan 👋 

Can you please verify that the user or db_name does not contain any typo or extra space? You may also want to restart your project on Supabase as well..

**Comment by kaelianbaudelet** [comment]
I know the problem. I have this problem when I don't choose a server in Virginia, America by default on Supabase. I have this problem frequently when I make a mistake and put the EU Paris server on Supabase, I'd have to check with Supabase support to have this problem corrected.
